src/Neo/Network/P2P/Payloads/TransactionAttributeType.cs

```diff
@@ -33,6 +33,12 @@ public enum TransactionAttributeType : byte
         /// Indicates that the transaction is not valid before <see cref="NotValidBefore.Height"/>.
         /// </summary>
         [ReflectionCache(typeof(NotValidBefore))]
-        NotValidBefore = 0x20
+        NotValidBefore = 0x20,
+
+        /// <summary>
+        /// Indicates that the transaction is a reserved attribute.
+        /// </summary>
+        [ReflectionCache(typeof(ReservedAttribute))]
+        ReservedAttribute = 0x31
```

Comment:
Actually, the idea was to have a _range_ of attributes (suppose you're creating an alternative to #1573 and defining 10 attributes instead of 3). And it's better be some upper range to reduce collisions with "regular" type.

Reply:
Like in https://github.com/neo-project/neo/issues/1904#issuecomment-715170305

Reply:
What do you mean by "Range of Attributes"?

In the way it is implemented we can setup a reserved byte that have `ff` possibilities as range for the dapp, right @shargon ?